src/modules/overview/screens/Overview/Overview.tsx

```diff
@@ -53,11 +53,18 @@ export const Overview = () => {
     dispatch(fetchPopularBrands());
   }, [dispatch]);
 
+  const createPromoSkeleton = () => {
+    return <ThemeProvider theme={darkTheme}>
+      <PromoSkeleton />
+    </ThemeProvider>
+  };
```

Comment:
```suggestion
  const renderedPromoSkeleton = (
    <ThemeProvider theme={darkTheme}>
      <PromoSkeleton />
    </ThemeProvider>
  );
```
Why are you using a function and not just a const?